### ISSUE TITLE
out of date api

### ISSUE THREAD

**Issue by LeonLIU08** [WeidiXie/New_Layers-Keras-Tensorflow#5]
I found your code -"normalized_correlation_layer.py" contains an import problem.

line 16: the conv_output_length function move to "keras.utils.conv_utils" rather than ".np_utils" in current version of keras.

Thank you.

**Comment by WeidiXie** [comment]
Sorry, this is too old repo, I am not maintaining it any more.
